financial-services-accelerator/react-apps/self-care-portal/self-care-portal-frontend/README.md

```diff
@@ -0,0 +1,24 @@
+# Self care portal application structure
+
+**accelerator folder**: Includes the code for the default consent manager application.
+
+**webpack.config.js**: Contains configurations for webpack.
+
+**loader.js**: A custom webpack loader to override default application to use toolkit specific application.
+
+## Setup
+
+Navigate to the **self-care-portal-frontend** directory in the terminal and run the following command.
+
+>npm install
+
+Existing components can be modified and/or new components can be added as well. For more details follow the 
+documentation [here.](https://ob.docs.wso2.com/en/latest/develop/customize-consent-manager-ui/)
```

Comment:
```suggestion
documentation [here.](https://ob.docs.wso2.com/en/latest/develop/customize-consent-manager-portal/
)
```

Reply:
Fixed in [3b64afd](https://github.com/wso2/financial-services-accelerator/pull/530/commits/3b64afda7a3f6a641860c1d09b54f01803bba706)